### DISCUSSION TITLE
function "polytope"

### DISCUSSION THREAD

**Discussion by Oyak1** [mtex-toolbox/mtex#767] [Ask Anything]
Dear MTEX team,
my system administrator recently updgraded my MATLAB version from R2011a to R2011b. Today I ran an old script that ususally worked with MTEX3.0. However, there was a problem with the function "polytope":
```
DATA(i) = polytope(grains(i));
XYdata = get(DATA(i),'xy');
```
"Undefined function 'polytope' for input arguments of type 'grain'."

The variable 'grains' is previously created using an MTEX grain reconstruction. I usually used that code to extract the xy-coordinates of grain polygones. Can you give me a hint how to modify to code to make it work again with MTEX3.0 in connection with MATLAB R2011b?

Sorry to bother you with questions about ancient versions...

Thanks for your help!
Oyak

**Comment by ralfHielscher** [comment]
Hi guess the easiest way for you would be to ask your system administrator to downgrade to Matlab 2011a.
The other option is to update Matlab to something more recent - maybe 2020b and update MTEX to 5.6.
This is much work I know. But believe me its worth.

Ralf.

**Reply by Oyak1** [reply]
Okay, you are probably right. I will start updating my code. Does MTEX 5.6 also work with MTALAB R2015a? This is the freshest version I have available.
Regards

**Comment by ralfHielscher** [comment]
You might try but I'm afraid you need something like 2016b - this is the
oldest I'm able to check.

Ralf.


********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************

**Comment by lfgmorales** [comment]
﻿Dear Oyak

A student license for matlad costs 100 dollars, I am fully aware that in many places this is a lot of money, but you may want to have a look here


https://www.mathworks.com/store/link/products/student/SV?s_tid=ac_buy_sv_but1

All the best

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 12 Feb 2021, at 08:28, Ralf Hielscher <notifications@github.com> wrote:

﻿

You might try but I'm afraid you need something like 2016b - this is the
oldest I'm able to check.

Ralf.


********************************************************************
Ralf Hielscher Tel: +371-531-38556
Fakultät für Mathematik +371-531-22200 (Sekr.)
Technische Universität Chemnitz Fax: +371-531-22109
Reichenhainer Str. 39 E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz http://www.tu-chemnitz.de/~rahi
********************************************************************


On Fri, Feb 12, 2021 at 8:25 AM Oyak1 <notifications@github.com> wrote:

> Okay, you are probably right. I will start updating my code. Does MTEX 5.6
> also work with MTALAB R2015a? This is the freshest version I have available.
> Regards
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mtex-toolbox/mtex/discussions/767#discussioncomment-361879>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACBYJ2L4LJJJDEORE37B2VLS6TJVRANCNFSM4XPLIO6Q>
> .
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/767#discussioncomment-361884>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZ72LJLGZTCFINSS23S6TKAXANCNFSM4XPLIO6Q>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/767#discussioncomment-361884", "url": "https://github.com/mtex-toolbox/mtex/discussions/767#discussioncomment-361884", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
